test/dymaptic.GeoBlazor.Core.Test.Blazor.Shared/Components/LayerTests.razor

```diff
@@ -0,0 +1,27 @@
+﻿@inherits TestRunnerBase
+
+@{
+    base.BuildRenderTree(__builder);
+}
+
+@code {
+
+    [TestMethod]
+    public async Task TestCanRenderKMLLayerWithUrl()
+    {
+        CreateViewRenderedHandler(async () =>
+    {
+        await AssertJavaScript("assertKmlLayerExists");
+    });
```

Comment:
indent formatting is a bit wonky

Reply:
I think this was a copy paste oversight when bringing this file in after the new test folder was created.  good to go